packages/common/src/components/Table/ExpandableTableRow.tsx

```diff
@@ -32,11 +21,7 @@ const ExpandableTableRow: React.FC<Props> = ({ rowContent, id, handleClick, chil
           if (idx === 0) {
             return (
               <td key={idx}>
-                {isExpanded ? (
-                  <ArrowUp src={arrow} alt="" onClick={toggleExpand} className="iu-mr-300" data-testid="expandArrow" />
-                ) : (
-                  <ArrowDown src={arrow} alt="" onClick={toggleExpand} className="iu-mr-300" data-testid="expandArrow" />
-                )}
+                <ArrowToggle onClick={handleToggle} className="iu-mr-200" label="Fäll ut/in tabellrader" isUp={isExpanded} />
```

Comment:
I think you should change the label "Fäll ut/in tabellrader" since it refers to the semantic structure. Perhaps "Fäll ut/in innehåll" might be more appropriate? You might actually have that as a standard label in `ArrowToggle` if no label prop is passed to it.